### ISSUE TITLE
Still maintained?

### ISSUE THREAD

**Issue by SpeedyCraftah** [nodemailer/smtp-server#211]
Hello,
I haven't seen much activity when it comes to this repository, and I know quite a lot of developers rely on it at the moment.
I have some features I'd like to add including some bug fixes, but I'm not sure if my changes will be merged, and I want to avoid making a fork and adding those fixes and additions myself.
What's the status of things at the moment?

One of the feature additions I had in mind at the moment is the ability to change the EHLO response in server options:
> S: 250-XYZ.org Nice to meet you, mail-il1-f182.google.com

The server response to the EHLO is not standardised with the exception of the status, server domain, and incoming domain, and so I think it would be a nice touch to allow the customisation of that from "Nice to meet you", to something else, especially since the project allows the addition of a custom banner as it is.

**Comment by endreszabo** [comment]
well, it is in `lib/smtp-connection.js`, that you can easily edit. :)

**Comment by SpeedyCraftah** [comment]
> well, it is in `lib/smtp-connection.js`, that you can easily edit. :)

Of course, but it's not that big of a feature to fork or modify the library and making any future updates very difficult to work with, just a nice thing to have, besides it wouldn't be the only thing I'd add or fix.

**Comment by SpeedyCraftah** [comment]
I have created a fork of smtp-server for developers that are still using it!

I will try to fix existing issues and add new features to keep up with modern SMTP servers, including the addition of new extensions and functionality, maybe even the ability to automatically parse incoming mail instead of having to use a separate library.

https://github.com/SpeedyCraftah/smtp-server
